### ISSUE TITLE
now option gives incorrect output

### ISSUE THREAD

**Issue by theycallmemac** [theycallmemac/dcurooms#22]
First experienced this issue on the 19/2/2018 at about 14:19.

A command such as: ```dcurooms -anL```, when ran at this time, should give output:
```
LG25: 14:00 ->
LG26: 14:00 ->
LG27: 14:00 ->
L125: 14:00 ->
L128: 14:00 ->
```

I know this should be the returned output as when I run ```dcurooms -lL 23 1 1400``` I get:
```
LG25: 14:00 ->
LG26: 14:00 ->
LG27: 14:00 ->
L101: 14:00 -> CA229[2]P1/01
Developing Internet Applications
L114: 14:00 -> CA229[2]P1/01
Developing Internet Applications
L125: 14:00 ->
L128: 14:00 ->
```

This shows the rooms LG25, LG26, LG27, L125 and L128 to be free when provided the academic week number, day number and timeslot for a lookup of the entire building.

Instead, my output looks something like:
```
LG27: 16:30 ->
L125: 16:30 ->
```
Given the ```16:30``` I figured I would look into it further by running ```dcurooms 23 1 1630```.
That returned the following output:

```
LG25: 16:30 -> CA269(2) P1/01
Computer Programming 4 (Object Orientated Programming)
LG26: 16:30 -> CA269(2) P1/01
Computer Programming 4 (Object Orientated Programming)
LG27: 16:30 ->
L101: 16:30 -> CA229[2]P1/01
Developing Internet Applications
L114: 16:30 -> CA229[2]P1/01
Developing Internet Applications
L125: 16:30 ->
L128: 16:30 -> CA269(2) P1/01
Computer Programming 4 (Object Orientated Programming)
```

This indeed shows both LG27 and L125 to be free.

The command ```dcurooms -anL``` is obviously having problems with the -n option over anything else. This is determined is it has been shown above that both -L and -a are working perfectly fine.

This needs fixing as it is an important feature of the program. For this issue to be be considered closed I'll require both:

- The code in [scripts/now.py](https://github.com/theycallmemac/dcurooms/blob/master/scripts/now.py) to be fixed.
- A new tests in [tests](https://github.com/theycallmemac/dcurooms/tree/master/tests) to runs checks on either -anL or -anC against the current time.

**Comment by theycallmemac** [comment]
Closed by #24

**Comment by theycallmemac** [comment]
Seems the issue has cropped up again, despite passing tests. Needs to be looked at once again.
